src/neo/SmartContract/Native/PolicyContract.cs

```diff
@@ -12,14 +10,9 @@ public sealed class PolicyContract : NativeContract
     {
         public const uint DefaultExecFeeFactor = 30;
         public const uint DefaultStoragePrice = 100000;
-        private const uint MaxExecFeeFactor = 1000;
-        private const uint MaxStoragePrice = 10000000;
```

Comment:
Why this two parameters be moved too? It seems that it has nothing to do with consensus.

Reply:
They are just two max limits to settings.